zerver/openapi/zulip.yaml

```diff
@@ -22539,6 +22594,23 @@ components:
               "result": "error",
               "var_name": "content",
             }
+    RestErrorHandlingSchema:
```

Comment:
I think it'd be better to have a schema for the generic error response that our API returns with the `code`, `msg` and `code` fields. And we create specific examples for particular `code` or `msg` responses.

And then more specialized errors, with additional fields, are still separate schema with their own examples.

That way we can potentially document the general error schema for all endpoints (for validation purposes), but possibly not always document an error for each endpoint response, unless it's necessary.